services/api/app/api/__init__.py

```diff
@@ -1,12 +0,0 @@
-"""create_app"""
-from fastapi import FastAPI
-
-
-def create_app() -> FastAPI:
-    app = FastAPI()
-
-    @app.get("/")
-    def root():
-        return {"message": "Hello World"}
-
-    return app
```

Comment:
I like the app factory structure here...keep this maybe? Tiangolo has some great docs about using routers (which are kinda like flask blueprints): https://fastapi.tiangolo.com/tutorial/bigger-applications/